importer/forms.py

```diff
@@ -6,31 +6,28 @@
 
 import os
 from tempfile import mkstemp
+import logging
 
-from django.forms import Form, FileField, ChoiceField
+from django.forms import Form, FileField
 
-from learningresources.models import Repository
 from importer.api import import_course_from_file
 
+log = logging.getLogger(__name__)
+
 
 class UploadForm(Form):
     """
     Form for allowing a user to upload a video.
     """
     course_file = FileField()
-    repository = ChoiceField(required=False)
 
     def __init__(self, *args, **kwargs):
         """
         Fill in choice fields.
         """
-        repos = Repository.objects.all().values_list('id', 'name')
         super(UploadForm, self).__init__(*args, **kwargs)
 
-        options = (('', '-- SELECT --'),) + tuple(repos)
-        self.fields['repository'].options = options
-
-    def save(self, user):
+    def save(self, user, repo_id):
         """
         Receives the request.FILES from the view.
 
```

Comment:
The next thing we're going to do is add async calls, and passing around primary keys is better than passing around objects for various reasons. Also, I want to prevent people from having to instantiate objects when only the pk is going to be used anyway, which it is in these cases.